client/src/test/helper.ts

```diff
@@ -11,14 +11,15 @@ export let platformEol: string;
  */
 export async function activate(docUri: vscode.Uri) {
     // The extensionId is `publisher.name` from package.json
-    const ext = vscode.extensions.getExtension('pvr.omt-odt-language')!;
-    await ext.activate();
-    try {
-        doc = await vscode.workspace.openTextDocument(docUri);
-        editor = await vscode.window.showTextDocument(doc);
-        await sleep(2000); // Wait for server activation
```

Comment:
The `sleep` function is no longer used now, can it go?

Reply:
this one should be gone already